ports/yaml-cpp/portfile.cmake

```
@@ -14,6 +14,12 @@ else()
     set(YAML_BUILD_SHARED_LIBS OFF)
 endif()
 
+if (NOT(WIN32) AND ${YAML_BUILD_SHARED_LIBS})
+    vcpkg_apply_patches(
+       SOURCE_PATH ${SOURCE_PATH}
+       PATCHES jbeder_yaml-cpp_958.patch)
+endif()
```

```suggestion
if (NOT WIN32 AND YAML_BUILD_SHARED_LIBS)
    vcpkg_apply_patches(
       SOURCE_PATH ${SOURCE_PATH}
       PATCHES jbeder_yaml-cpp_958.patch)
endif()
```
Seems that `jbeder_yaml-cpp_958.patch` is missing in this PR. So I wonder where do you get it now?